### PR TITLE
[#1133] Remove Edit Buttons From Collection and Generic File Views

### LINKED ISSUE #1133: Remove Edit Buttons From Collection and Generic File Views
Remove the edit buttons for Collections and Generic Files on DigitalHub. This will **only** change the views so check for ways that a more advanced user might get around this in case it is something that we decide is worth closing down as well.

Test on staging to confirm everything is working. After opening the PR for this work **do not merge**. Much like disabling the file upload this will be merged at a later date.

### PR DESCRIPTION
**Pull request by Meowcenary**
Remove buttons that can change collections and generic files from all views. closes #1133

### CODE REVIEW

**Comment by Meowcenary** [comment]
This will be merged whenever we decide to close DigitalHub to editing.

File: app/views/my/_action_menu.html.erb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <%= link_to raw('<i class="glyphicon glyphicon-download-alt" aria-hidden="true"></i> Download File'), sufia.download_path(id),
+      class: 'itemicon itemdownload', title: 'Download File', target: '_new' %>
+    </li>
+    <li role="menuitem" tabindex="-1">
+      <%= display_trophy_link(@user, id) do |text| %>
+        <i aria-hidden="true" class='glyphicon glyphicon-star'></i> <%= text %>
+      <% end %>
+    </li>
+  </ul>
+</div>
```
Comment (fenekku) [inline app/views/my/_action_menu.html.erb:22]: Is this a template to override another a default one where any mention of create/edit/delete has been removed?

**Review by fenekku — APPROVED** [review]
All of those make sense to me. Like you said it's going to be a matter of visiting the site and seeing if we missed any. This could probably be deployed on prototype for that purpose.

File: app/views/my/_action_menu.html.erb (hunk 1 above)
Comment (Meowcenary) [reply app/views/my/_action_menu.html.erb:22]: Yes, exactly that. To find the default templates I've been referencing this branch of the Sufia project: https://github.com/samvera-deprecated/sufia/tree/6.x-stable

**Comment by sharpattack** [comment]
@fenekku I think this is deployed to staging, right @Meowcenary? 

Having some trouble with our dummy student netid(I think it's expired), but as myself, I can still visit /edit on any file and make changes. Can a normal user do the same?

**Comment by Meowcenary** [comment]
> 

I'll have to check my roles, but i'm logged in on staging and unable to edit or delete any of my files/collections or any other that I'm coming across.

**Comment by carsonicator** [comment]
@sharpattack were you using the gso1135 NetID? I will check with IT and reactivate if expired.

**Review by carsonicator — APPROVED** [review]
Thanks @Meowcenary

**Comment by sharpattack** [comment]
@carsonicator I am having trouble with both netids. I just opened a ticket with FSMIT and CC'd the galter-is email.

**Comment by Meowcenary** [comment]
I'm going to give myself more roles on staging to see if that reveals edit buttons I was not seeing previously. Maybe there's something like that going on here.

**Comment by Meowcenary** [comment]
Aha - batch editing buttons only appear after you've clicked a checkbox for a file. I was only looking at the actions dropdown for the files.

**Comment by Meowcenary** [comment]
Going to close this out now that #1117 has replaced #1133 .

**Pull request closed without merging**